application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,36 @@
+function ErrorSummaryWithStickyBanner() {
+}
+
+ErrorSummaryWithStickyBanner.prototype.init = function () {
+    var hrefAnchorRegex = /\#(.+)$/;
```

Comment:
use `\z` for end-of-string instead of `$` for end-of-line? Unlikely to make much difference, unless we start putting line breaks in mid-attribute, though.

Reply:
I think `\z` might be a ruby-ism? I've tried it in two JS regex things online and it breaks, and no reference to it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)?

Reply:
Ah! Did not know that...